### ISSUE TITLE
Deprecate the accessibility mode setting

### ISSUE THREAD

**Issue by jasonwebb** [Accessible360/accessible-slick#12]
Accessibility should not be something that can be turned on or off with a feature flag. Instead it is the natural result of building clean, modern, and semantic code, and should be considered a necessary core feature.

- [x] Remove all conditionals using the `accessibility` variable.
- [x] Remove the `initADA` and `activateADA` methods, integrating any important functionality directly into the core functionality.
  - Besides, none of this functionality is explicitly called out by the ADA (or Section 508, or AODA, or any other civil rights legislation around equitable access). Instead, this legislation (and related case law) leans on the WCAG as a technical criteria. 
  - At the very least, this functionality is about basic human equality, which we shouldn't really need a law to make us do. These method names seem to imply that the important thing is that a legal requirement is being targeted and met, when really it should just be something we do automatically.
- [x] Once this functionality is removed, ensure that passing the `accessibility` setting via a config or data attribute does not introduce any bugs.
- [x] Consider emitting a console warning or info message when this setting is passed to alert devs about the change in functionality.
- [x] Add documentation of this change to the main README.
